.github/workflows/ci.yaml

```diff
@@ -5,55 +5,55 @@
 
 name: CI
 "on":
-  pull_request:
-    branches:
-    - '*'
-    paths-ignore:
-    - README.md
-  push:
-    branches:
-    - main
-    paths-ignore:
-    - README.md
+    push:
```

Comment:
Not really that important but I wouldn't mind if we would go back to 2 spaces.

Reply:
Done.

Reply:
Still some intendation changes but way less. Thank you.

Reply:
The indentation changes that you still see are the new list format.

Old:
```yaml
object:
- one
- two
```

New:
```yaml
object:
  - one
  - two
```

IMO, the new format looks nice as it shows the list objects nested under the parent.

Reply:
Yeah, I also like the new one better. Also I don't think it would be easy or worth our time to revert that.